app/src/main/AndroidManifest.xml

```diff
@@ -2,78 +2,73 @@
             android:name=".view.MainActivity"
-            android:screenOrientation="portrait">
-        </activity>
+            android:launchMode="singleInstance"
+            android:screenOrientation="portrait"></activity>
         <activity
             android:name=".view.SplashActivity"
             android:screenOrientation="portrait"
             android:theme="@style/SplashTheme">
             <intent-filter>
-                <action android:name="android.intent.action.MAIN"/>
-                <category android:name="android.intent.category.LAUNCHER"/>
+                <action android:name="android.intent.action.MAIN" />
+                <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
 
         <activity
             android:name=".view.earn.QuestionnaireActivity"
             android:screenOrientation="portrait"
-            android:theme="@style/Questionnaire"/>
+            android:theme="@style/Questionnaire" />
 
         <activity
             android:name=".view.earn.WebTaskCompleteActivity"
             android:screenOrientation="portrait"
-            android:theme="@style/Questionnaire"/>
+            android:theme="@style/Questionnaire" />
 
 
         <activity
             android:name=".view.tutorial.TutorialActivity"
-            android:screenOrientation="portrait"
-           />
+            android:screenOrientation="portrait" />
 
         <activity
             android:name=".view.phoneVerify.PhoneVerifyActivity"
             android:screenOrientation="portrait"
-            android:windowSoftInputMode="adjustResize"
-            />
+            android:windowSoftInputMode="adjustResize" />
 
         <activity
             android:name=".view.spend.PurchaseOfferActivity"
-            android:screenOrientation="portrait"
-            />
-
+            android:screenOrientation="portrait" />
 
 
         <activity
             android:name=".view.spend.Peer2PeerActivity"
             android:screenOrientation="portrait"
-            android:windowSoftInputMode="adjustResize"
-            />
-        <activity android:name=".view.earn.WebTaskActivity"
-            android:screenOrientation="landscape"/>
+            android:windowSoftInputMode="adjustResize" />
+        <activity
+            android:name=".view.earn.WebTaskActivity"
+            android:screenOrientation="landscape" />
 
         <service android:name=".network.firebase.KinInstanceIdService">
             <intent-filter>
-                <action android:name="com.google.firebase.INSTANCE_ID_EVENT"/>
+                <action android:name="com.google.firebase.INSTANCE_ID_EVENT" />
             </intent-filter>
         </service>
         <service android:name=".network.firebase.KinMessagingService">
             <intent-filter>
-                <action android:name="com.google.firebase.MESSAGING_EVENT"/>
+                <action android:name="com.google.firebase.MESSAGING_EVENT" />
```

Comment:
Has something changed on this file or just formatting?

Reply:
Auto (on save) Linting.